raiden/transfer/channel.py

```diff
@@ -1705,6 +1713,10 @@ def handle_channel_closed(
         )
         if call_update:
             expiration = state_change.block_number + channel_state.settle_timeout
+            # silence mypy: checked above already
+            assert balance_proof is not None
```

Comment:
Hmmm this is a double assertion. Right above inside `call_update` we are checking that `balance_proof` is not None. Please find a way to not do this twice.